helm/airflow/templates/rbac/pod-launcher-rolebinding.yaml

```diff
@@ -49,6 +49,9 @@ roleRef:
 {{- end }}
   name: {{ .Release.Name }}-pod-launcher-role
 subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}-webserver
+    namespace: {{ .Release.Namespace }}
```

Comment:
@pbastia this might be worth contributing upstream, or at least creating an issue about it